_layouts/archive.html

```diff
@@ -1,27 +1,70 @@
 ---
 layout: default
 ---
-<article class="archive">
+
+{% assign collection = site.collections | where: "title", page.title | first %}
+
+<article class="archive archive--{{ page.title | slugify }}" data-collection="{{ collection.title }}"  role="content" itemscope itemtype="http://schema.org/BlogPosting" {{ data_facet }} {{ data_facet_value }}>
 
   {% include page-header.html %}
 
   <section class="archive__filter-type">
-    Filter by Type
+
+    {% if page.permalink == '/issue-briefs/' %}
+      <div id="filter__brief-clear"></div>
+      <div class="all-result-count-placeholder"> </div>
+      <div id="filter__content-brief"></div>
+    {% endif %}
   </section>
+
   <section class="archive__results">
-    Filter Results<br />
-    Sort
+    <div id="results__summary"></div>
+    <div id="results__sort">
+      <span class="sort-oldest">OLDEST</span>
+      <span class="sort-newest selected">NEWEST</span>
+    </div>
   </section>
+
   <section class="archive__sidebar">
-    Refine Results
+      {% if page.permalink == '/search/' or page.permalink == '/resources/' or page.permalink == '/analysis/' %}
```

Comment:
A cleaner way to do this would be to assign a front matter boolean variable in the front matter of each of these pages. Then you only have to check if page.variable and not multiple permalinks.